src/pykiso/config_parser.py

```diff
@@ -162,20 +162,35 @@ def _fix_types_loc(thing: dict) -> dict:
         return thing
 
     def _parse_env_var(config_element: str) -> str:
-        """Search for an environment variable and replace it with the associated value.
+        """Search for an environment variable and replace it with the associated value if it find one.
+        Otherwise use the default value if one is specified.
 
         Additionally, cast the value if it matches an integer.
 
         :param config_element: config sub-dict value
 
         :return: config sub-dict value with replaced environment variable if needed
         """
+        has_default_value = False
+        # Handle default value
+        if all(element in config_element for element in ["ENV", "="]):
+            default_value = re.search("=(.*)}", config_element).group(1)
+            config_element = config_element[: config_element.find("=")] + "}"
+            has_default_value = True
         # Check for regular expression "ENV{word}"
         match = re.compile(r"ENV{(\w+)}").findall(str(config_element))
         if match:
```

Comment:
I'm surprised that match contains something and that you go in

Reply:
It does, but it is much simpler to do as you said above